src/Datadog.Trace/Metrics.cs

```diff
@@ -3,5 +3,13 @@ namespace Datadog.Trace
     internal static class Metrics
     {
         public const string SamplingPriority = "_sampling_priority_v1";
+
+        public const string SamplingAgentDecision = "_dd.agent_psr";
```

Comment:
What is `psr`? Per sample rate?

Reply:
I don't actually know for sure, I assumed "priority sample rate", but I'll need to verify.